frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -114,7 +120,12 @@ export const Router: React.FC = () => {
     <ClientProvider>
       <PageContainer
         menuItems={menuItems}
-        contentRoutes={routes(talentPaths, profilePaths)}
+        contentRoutes={[
+          ...talentRoutes(talentPaths),
+          ...(isStringTrue(useRuntimeVariable("FEATURE_APPLICANTPROFILE"))
```

Comment:
I would define a helper function which combines isStringTrue with useRuntimeVariable for you, since the use of isStringTrue is not totally obvious. eg
```
checkFeatureFlag = (name: string): boolean => {
   // We use isStringTrue here because an ENV variable which is left undefined may show up as a runtime variable as a string (eg "false" or "(none)") instead of false or undefined.
    return isStringTrue(useRuntimeVariable(name));
 }
```

Reply:
Good call, I've added in the checkFeatureFlag help function.